Forms/GridField/GridField.php

```diff
@@ -307,12 +307,12 @@ public function getState($getData = true) {
 	 */
 	public function FieldHolder($properties = array()) {
 		Requirements::css(THIRDPARTY_DIR . '/jquery-ui-themes/smoothness/jquery-ui.css');
-		Requirements::css(FRAMEWORK_DIR . '/client/dist/styles/GridField.css');
+		Requirements::css(trim(FRAMEWORK_DIR . '/client/dist/styles/GridField.css', '/'));
```

Comment:
Also, once we move to code-outside-of-webroot, this is going to need to be changed anyway. The fact that it's currently ugly is temporary. :)